src/ansys/hps/client/jms/api/project_api.py

```diff
@@ -593,16 +601,18 @@ def delete_license_contexts(self):
         """Delete license contexts."""
         rest_name = LicenseContext.Meta.rest_name
         url = f"{self.jms_api_url}/projects/{self.id}/{rest_name}"
-        r = self.client.session.delete(url)
+        _ = self.client.session.delete(url)
 
     ################################################################
     def copy_default_execution_script(self, filename: str) -> File:
         """Copy a default execution script to the current project.
 
         Example:
 
-            >>> file = project_api.copy_default_execution_script("exec_mapdl.py")
+            >>> file = project_api.copy_default_execution_script("mapdl-v241-exec_mapdl.py")
 
+        You can use the method :meth:`JmsApi.list_default_execution_scripts` to query
```

Comment:
```suggestion
        You can use the :meth:`JmsApi.list_default_execution_scripts` method to query
```
Noun indicating the type of object should follow the tagged code entity.